### PR TITLE
Add _dynamo.config.suppress_errors logging

### PR DESCRIPTION
**Pull request by jovianjaison**
cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang @amjames @rec

### CODE REVIEW

File: torch/_dynamo/convert_frame.py [hunk 1]
```diff
@@ -1062,6 +1062,7 @@ def format_guard_failures() -> str:
                 possibly_missed_reinplacing_opportunities,
                 remote_cache_time_saved,
                 structured_logging_overhead_s,
+                config.suppress_errors
```
Comment (ezyang) [inline torch/_dynamo/convert_frame.py:1065]: trailing comma

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
What's the companion logger update diff? Link it here

**Comment by jovianjaison** [comment]
> What's the companion logger update diff? Link it here

[D62765788]

File: torch/_dynamo/convert_frame.py (hunk 1 above)
Comment (jovianjaison) [reply torch/_dynamo/convert_frame.py]: done

**Pull request closed without merging**